spec/acceptance/dispatch_router_spec.rb

```diff
@@ -5,28 +5,7 @@
     it_behaves_like 'an idempotent resource' do
       let(:manifest) do
         <<-PUPPET
-      include foreman_proxy_content::dispatch_router
-        PUPPET
-      end
-    end
-
-    describe service('qdrouterd') do
-      it { is_expected.to be_running }
-      it { is_expected.to be_enabled }
-    end
-
-    describe port('5647') do
-      it { is_expected.to be_listening }
-    end
-  end
-
-  context 'with ensure absent' do
-    it_behaves_like 'an idempotent resource' do
-      let(:manifest) do
-        <<-PUPPET
-        class { 'foreman_proxy_content::dispatch_router':
-          ensure => 'absent',
-        }
+        class { 'foreman_proxy_content::dispatch_router': }
```

Comment:
Why change the previous line?
```suggestion
        include foreman_proxy_content::dispatch_router
```

Reply:
I think it's just a coincidence of the way I made the change in the editor vs. the way the diff got rendered. I changed 'with ensure absent' to 'with default parameters' and the next couple lines were the same.

To your point, I can change this to an include statement since we're no longer passing parameters here.